lib/nostrum/api.ex

```diff
@@ -734,15 +737,32 @@ defmodule Nostrum.Api do
   def bulk_delete_messages(channel_id, messages, filter \\ true)
 
   def bulk_delete_messages(channel_id, messages, false),
-    do: request(:post, Constants.channel_bulk_delete(channel_id), %{messages: messages})
+    do: send_chunked_delete(channel_id, messages)
 
   def bulk_delete_messages(channel_id, messages, true) do
     filtered_messages =
       Enum.filter(messages, fn message_id ->
         (message_id >>> 22) + 1_420_070_400_000 > Util.now() - 14 * 24 * 60 * 60 * 1000
       end)
 
-    request(:post, Constants.channel_bulk_delete(channel_id), %{messages: filtered_messages})
+    send_chunked_delete(channel_id, filtered_messages)
```

Comment:
If we rewrote `send_chunked_delete/2` to accept `filtered_messages` as the first parameters we could easily pipe all of this, IMO it would be much cleaner.

Reply:
This does fall out of line with how we do other functions in this module, but like I said elsewhere it should probably be private anyways.